packages/desktop/views/dashboard/account-management/AccountManagement.svelte

```diff
@@ -2,14 +2,18 @@
     import { selectedWallet } from '@core/wallet/stores'
     import features from '@features/features'
     import { AccountManagementDetails, AccountManagementList } from '@components'
-    import { OutputData } from '@iota/sdk/out/types'
+    import { AccountOutput, OutputData } from '@iota/sdk/out/types'
+    import { isAccountOutput, isImplicitAccountOutput } from '@core/wallet'
 
-    $: allAccountOutputs = [
-        ...($selectedWallet?.accountOutputs || []),
-        ...($selectedWallet?.implicitAccountOutputs || []),
-    ]
+    $: allAccountOutputs =
+        $selectedWallet?.walletUnspentOutputs?.filter(
+            (output) => isAccountOutput(output) || isImplicitAccountOutput(output)
+        ) || []
 
-    $: selectedOutput = allAccountOutputs?.[0]
+    let selectedOutput =
```

Comment:
any reason this isn't a reactive value anymore?

Reply:
It is the output that you have selected in the account list, if an account is activated (and `allAccountOutputs` is modified), I want to continue having the output that I have selected, not the main or the first one. wdyt?

Reply:
gotcha